effect/shared/src/main/scala/scalaz/effect/IOInstances.scala

```diff
@@ -18,4 +18,11 @@ trait IOInstances {
       override final def flatMap[A, B](ma: IO[E, A])(f: A => IO[E, B]): IO[E, B] =
         ma.flatMap(f)
     })
+
+  implicit final val bifunctor: Bifunctor[IO] =
+    instanceOf(new BifunctorClass[IO] with BifunctorClass.DeriveBimap[IO] {
```

Comment:
can be just `instanceOf(new BifunctorClass.DeriveBimap[IO] {`

Reply:
fixed 👍